tools/tuistbench/README.md

```diff
@@ -0,0 +1,161 @@
+# Tuist Benchmark
```

Comment:
Same comment here in regards to performance. I'd move it to the documentation website.